OSOE-46: ⭐ Use WebApplicationFactory directly instead of running the tested app with the dotnet CLI

dministro opened the pull request:
OSOE-46

dministro committed:
Adjusting UITestBase and related web project
Updating submodules:
 - Lombiq.UITestingToolbox: Injecting FakeStore

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Updating submodules:
 - Lombiq.HelpfulExtensions: update to dev
 - Lombiq.Privacy: update to dev
 - Lombiq.BaseTheme: update to dev
 - Lombiq.UITestingToolbox: Clearing SQLite connection pool

dministro committed:
Adding Lombiq.Tests.UI.Shortcuts to Lombiq.OSOCE.Tests recipe

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Updating submodules:
 - Lombiq.UITestingToolbox: Fixing CA1063: Provide an overridable implementation of Dispose(bool)

dministro committed:
Fixing SA1505: An opening brace should not be followed by a blank line

dministro committed:
Fixing the cause of failing tests:
 - BehaviorVueTests.RecipeDataShouldBeDisplayedCorrectly(browser: Chrome)
 - EmailTests.SendingTestEmailShouldWork(browser: Chrome)
Updating submodule:
 - Lombiq.UITestingToolbox: Fixing the cause of failing tests

dministro committed:
Updating submodule:
 - Lombiq.UITestingToolbox: Maybe fixing failing test: Lombiq.Tests.UI.Samples.Tests.ErrorHandlingTests.ErrorDuringSetupShouldHaltTest(browser: Chrome)

BenedekFarkas committed:
Experimental update to the build-and-test workflow

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Fixing GH workflow

dministro committed:
Fixing wf again

dministro committed:
Fixing GH wf

BenedekFarkas committed:
Fixing command in experimental change in build-and-test.yml

dministro committed:
Merge branch 'issue/OSOE-46' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

BenedekFarkas committed:
Updating submodules to match dev / merging from dev to issue branch

BenedekFarkas committed:
Merge branch 'dev' into issue/OSOE-46

dministro committed:
Merge branch 'issue/OSOE-46' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Updating submodule:
 - Lombiq.Hosting.Azure.ApplicationInsights: Changing to InstanceCommandLineArgs

dministro committed:
Updating submodules:
 - Lombiq.Hosting.Azure.ApplicationInsights: Fixing command line arg
 - Lombiq.UITestingToolbox: Fixing command line arg

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Updating submodule:
 - Lombiq.Hosting.Tenants: Changing to InstanceCommandLineArgumentsBuilder

dministro committed:
Updating submodules:
 - Lombiq.Hosting.Azure.ApplicationInsights: Renaming AddValue to AddWithValue
 - Lombiq.Hosting.Tenants: Renaming AddValue to AddWithValue
 - Lombiq.UITestingToolbox: Renaming AddValue to AddWithValue

dministro committed:
Removing dotnet-info task from GH workflow

dministro committed:
Changing to InstanceCommandLineArgumentsBuilder

dministro committed:
Updating submodule:
 - Lombiq.Hosting.Tenants: Increasing delay

dministro committed:
Updating submodule:
 - Lombiq.Hosting.Tenants: Increasing delay, last try

dministro committed:
Updating submodule:
 - Lombiq.Hosting.Tenants: Adding more logs

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Updating submodule:
 - Lombiq.Hosting.Tenants: Reverting log level changes

dministro committed:
Adding log dump
Updating submodule:
 - Lombiq.Hosting.Tenants: Removing log dump

dministro committed:
Removing extra log
Updating submodule:
 - Lombiq.Hosting.Tenants: Removing extra logs

dministro committed:
Disabling tests except IdleTenants
Adding appsettings.json
Updating submodule:
 - Lombiq.UITestingToolbox: Disabling tests

dministro committed:
Crossprobing appsettings.json

dministro committed:
Enabling BehaviorApplicationInsightsTests

dministro committed:
Enabling BehaviorBaseThemeTests

dministro committed:
Enabling BehaviorChartJsTests

dministro committed:
Enabling BehaviorHelpfulExtensionsTests

dministro committed:
Enabling BehaviorPrivacyTests

dministro committed:
Enabling BehaviorVueTests

dministro committed:
Enabling CustomAdminPrefixTests

dministro committed:
Enabling RetrievalDataTablesTests

dministro committed:
Adding appsettings.json

dministro committed:
Disabling IdleTenantsTests

dministro committed:
Removing appsettings.json

dministro committed:
Enabling IdleTenantsTests
Changing ui-test-parallelism to 1 in GH workflow

dministro committed:
Reverting ui-test-parallelism

dministro committed:
Enabling IdleTenantsTests again

dministro committed:
Updating submodule:
 - Lombiq.UITestingToolbox: Adding logger configuration

dministro committed:
Updating submodules:
 - Lombiq.Hosting.Tenants: Decreasing delay
 - Lombiq.UITestingToolbox: Enabling Lombiq.Tests.UI.Samples tests

dministro committed:
Updating submodule:
 - Lombiq.UITestingToolbox: Fixing MonkeyTests

dministro committed:
Updating submodule:
 - Lombiq.UITestingToolbox: Fixing MonkeyTests again

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Adjusting tests
Updating submodule:
 - Lombiq.UITestingToolbox: Using OC services directly instead of Shortcut Controllers where possible, the rest

dministro committed:
Updating submodules

dministro committed:
Updating submodule:
 - Lombiq.UITestingToolbox: Removing unused variable

dministro committed:
Updating submodules:
 - Lombiq.Privacy: Adjusting UI tests to new behavior
 - Lombiq.UITestingToolbox: Injecting fake HttpContext before calling OC services directly

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Updating submodule:
 - Lombiq.UITestingToolbox: Using OC services directly instead of Shortcut Controllers where possible, one more

dministro committed:
Adding consumer classes for test classes from Lombiq.Tests.UI.Samples
Updating submodule:
 - Lombiq.UITestingToolbox: Using Lombiq.Tests.UI.Samples test classes from Lombiq.OSOCE.Tests.UI as a consumer.

dministro committed:
Updating submodule:
 - Lombiq.UITestingToolbox: Removing test sdk from Lombiq.Tests.UI.Samples.csproj

dministro committed:
Migrating NuGetTest to new approach
Updating submodules:
 - Lombiq.ChartJs: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.DataTables: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.HelpfulExtensions: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Hosting.Azure.ApplicationInsights: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Hosting.Tenants: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Privacy: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.BaseTheme: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.UITestingToolbox: Adding Lombiq.Tests as a NuGet package

dministro committed:
Disabling sample tests

dministro committed:
Enabling BasicTests

dministro committed:
Reverting "Adding consumer classes for test classes from Lombiq.Tests.UI.Samples"

dministro committed:
Removing Lombiq.Tests.UI.AppExtensions reference from Lombiq.OSOCE.Web and Lombiq.OSOCE.NuGet.Web
Fixing comments
Updating submodules:
 - Lombiq.HelpfulExtensions: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.UITestingToolbox: Fixing docs

dministro committed:
Updating NuGet package versions
Updating submodules:
 - Lombiq.ChartJs: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.DataTables: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.HelpfulExtensions: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Hosting.Azure.ApplicationInsights: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Hosting.Tenants: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Privacy: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.BaseTheme: Updating Lombiq.Tests.UI.* Nuget package version

dministro committed:
Updating submodule:
 - Lombiq.UITestingToolbox: Removing possible dead-lock

dministro committed:
Updating NuGet package versions
Updating submodules:
 - Lombiq.ChartJs: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.DataTables: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.HelpfulExtensions: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Hosting.Azure.ApplicationInsights: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Hosting.Tenants: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Privacy: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.BaseTheme: Updating Lombiq.Tests.UI.* Nuget package version

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Merge branch 'issue/OSOE-46' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Updating NuGet package versions
Updating submodule:
 - Lombiq.UIKit: Fixing License.md and Readme.md settings in Lombiq.UIKit.Tests.UI.csproj

dministro committed:
Fixing UIKit NuGet package version

dministro committed:
Updating submodule:
 - Lombiq.UIKit: Fixing incompatible syntax

dministro committed:
Updating submodules:
 - Lombiq.UIKit: Revert "Fixing incompatible syntax"
 - Lombiq.UITestingToolbox: Fixing razor runtime compilation related issue

dministro committed:
Updating submodule:
 - Lombiq.UITestingToolbox: Just for a test

dministro committed:
Adding preserve compilation context to keep compilationOptions in .deps.json for razor runtime compilation
Update submodule:
 - Lombiq.UITestingToolbox: Removing unnecessary workaround for razor runtime compilation

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Updating NuGet package versions
Updating submodules:
 - Lombiq.ChartJs: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.DataTables: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.HelpfulExtensions: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Hosting.Azure.ApplicationInsights: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Hosting.Tenants: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Privacy: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.BaseTheme: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.UIKit: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.GitHub.Actions: Merge

dministro committed:
Disabling some tests in NuGt tests

dministro committed:
Enabling BehaviorApplicationInsightsTests in NuGet tests

dministro committed:
Enabling BehaviorChartJsTests in NuGet tests

dministro committed:
Enabling BehaviorDataTableTests in NuGet tests

dministro committed:
Enabling BehaviorFileUploadTests in NuGet tests

dministro committed:
Enabling BehaviorHelpfulExtensionsTests in NuGet tests

dministro committed:
Enabling BehaviorPrivacyTests in Nuget tests

dministro committed:
Enabling BehaviorThemeTests in NuGet tests

dministro committed:
Adjusting BaseThemeShouldWork
Updating submodule:
 - Lombiq.UITestingToolbox: Updating docs

dministro committed:
Disabling all tests except BehaviorThemeTests in NuGet tests

dministro committed:
Temporary disabling dotnet consolidation verification
Updating Lombiq.Tests.UI.* Nuget package version
Updating submodule:
 - Lombiq.UITestingToolbox: Possible fix of real deadlock Updating docs

dministro committed:
Reverting: "Temporary disabling dotnet consolidation verification"
Updating NuGet package versions
Updating submodules:
 - Lombiq.ChartJs: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.DataTables: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.HelpfulExtensions: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Hosting.Azure.ApplicationInsights: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Hosting.Tenants: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Privacy: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.BaseTheme: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.UIKit: Updating Lombiq.Tests.UI.* Nuget package version

dministro committed:
Updating submodules:
 - Lombiq.HelpfulLibraries: update to latest dev
 - Lombiq.OrchardCoreApiClient: update to latest dev
 - Lombiq.AuditTrailExtensions: update to latest dev
 - Lombiq.JsonEditor: update to latest dev
 - Lombiq.TrainingDemo: update to latest dev
 - Lombiq.Hosting.MediaTheme: update to latest dev
 - Lombiq.Gulp.Extensions: update to latest dev
 - Lombiq.Npm.Targets: update to latest dev
 - Lombiq.SetupExtensions: update to latest dev
 - Lombiq.UITestingToolbox: Adding IServiceProviderInstance to fake HttpContext

dministro committed:
Removing PreserveCompilationContext
Updating submodules:
 - Lombiq.UITestingToolbox: Trying to avoid adding razor runtime view compilation
 - Lombiq.GitHub.Actions: Merge

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

dministro committed:
Updating submodule:
 - Lombiq.Hosting.MediaTheme: Updating Lombiq.Tests.UI.* Nuget package version

dministro committed:
Removing PreserveCompilationContext from web apps
Updating submodules:
 - Lombiq.UITestingToolbox: Updating docs
 - Lombiq.Analyzers: Merge
 - Lombiq.Analyzers.PowerShell: Merge

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

# Conflicts:
#	src/Modules/Lombiq.Hosting.Tenants
#	src/Themes/Lombiq.Hosting.MediaTheme

Piedone commented on NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj:
```diff
@@ -10,27 +10,30 @@
   </ItemGroup>
 
+  <ItemGroup>
+    <InternalsVisibleTo Include="Lombiq.OSOCE.NuGet.Tests.UI" />
```
Do we actually need this in OSOCE? Since no non-public type will be accessed by the tests.

sarahelsaig commented on NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Program.cs:
```diff
@@ -7,14 +7,25 @@
 builder.Host.UseNLogHost();
 
 var configuration = builder.Configuration;
+builder.Services.Add(new ServiceDescriptor(configuration.GetType(), configuration));
```
Isn't this the same as `builder.Services.AddSingleton(configuration);`? Same [here](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/blob/050499fc48c31b04e0d4b70e676e8edd8860e640/src/Lombiq.OSOCE.Web/Program.cs#L14).

sarahelsaig commented on NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Program.cs:
```diff
@@ -7,14 +7,25 @@
 app.UseOrchardCore();
 app.Run();
+
+// As described here(https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-6.0).
+#pragma warning disable CA1050
+public partial class Program
+#pragma warning restore CA1050
```
I think using `[SuppressMessage]` would be cleaner for suppressing. Same [here](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/blob/050499fc48c31b04e0d4b70e676e8edd8860e640/src/Lombiq.OSOCE.Web/Program.cs#L24-L27).

sarahelsaig commented on NuGetTest/test/Lombiq.OSOCE.NuGet.Tests.UI/Lombiq.OSOCE.NuGet.Tests.UI.csproj:
```diff
@@ -16,16 +16,20 @@
-    <PackageReference Include="Lombiq.Privacy.Tests.UI" Version="3.1.3-alpha.walma-15.2" />
-    <PackageReference Include="Lombiq.HelpfulExtensions.Tests.UI" Version="3.5.1-alpha-osoe-185.0" />
-    <PackageReference Include="Lombiq.Tests.UI" Version="3.5.1-alpha.walma-15.4" />
+    <PackageReference Include="Lombiq.BaseTheme.Tests.UI" Version="1.1.3-alpha.osoe-46.4" />
+    <PackageReference Include="Lombiq.ChartJs.Tests.UI" Version="2.3.2-alpha.osoe-46.4" />
+    <PackageReference Include="Lombiq.DataTables.Tests.UI" Version="3.2.2-alpha.osoe-46.4" />
+    <PackageReference Include="Lombiq.Hosting.Azure.ApplicationInsights.Tests.UI" Version="3.2.1-alpha.osoe-46.4" />
+    <PackageReference Include="Lombiq.Hosting.Tenants.IdleTenantManagement.Tests.UI" Version="2.2.1-alpha.osoe-46.4" />
+    <PackageReference Include="Lombiq.Privacy.Tests.UI" Version="3.1.3-alpha.osoe-46.4" />
+    <PackageReference Include="Lombiq.HelpfulExtensions.Tests.UI" Version="3.5.1-alpha.osoe-46.4" />
+    <PackageReference Include="Lombiq.Tests.UI" Version="3.5.1-alpha.osoe-46.5" />
```
Are these actually necessary? I think these packages are already referenced indirectly through `..\..\src\Lombiq.OSOCE.NuGet.Web\Lombiq.OSOCE.NuGet.Web.csproj`.

dministro replied:
Only the `*.Tests.UI` packages are added here. Module and module sample packages are added to `Lombiq.OSOCE.NuGet.Web.csproj`. So yes, these are neccessary.

dministro committed:
Removing unnecessary InternalsVisibleTo
Using AddSingleton<>
Updating submodules:
 - Lombiq.ChartJs: Merge...
 - Lombiq.DataTables: Merge...
 - Lombiq.HelpfulExtensions: Merge...
 - Lombiq.Hosting.Azure.ApplicationInsights: Merge...
 - Lombiq.Hosting.Tenants: Merge...
 - Lombiq.Privacy: Merge...
 - Lombiq.UIKit: Merge...
 - Lombiq.BaseTheme: Merge...
 - Lombiq.Tests: Update...
 - Lombiq.UITestingToolbox: Merge..
 - Lombiq.GitHub.Actions: Merge...

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

# Conflicts:
#	NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj
#	NuGetTest/test/Lombiq.OSOCE.NuGet.Tests.UI/Lombiq.OSOCE.NuGet.Tests.UI.csproj
#	src/Themes/Lombiq.Hosting.MediaTheme

dministro committed:
Temprary disabling NuGet build and test
Updating submodules:
 - Lombiq.Hosting.Azure.ApplicationInsights: Using boolean instead of string in command line
 - Lombiq.Tests: Using Uri instead of string
 - Lombiq.UITestingToolbox: Updating docs Adding comments Adding InstanceCommandLineArgumentsBuilder.Add with Obsolate attribute

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

# Conflicts:
#	NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj
#	NuGetTest/test/Lombiq.OSOCE.NuGet.Tests.UI/Lombiq.OSOCE.NuGet.Tests.UI.csproj

dministro committed:
Temprary disabling verify-dotnet-consolidation

dministro committed:
Reverting: "Temprary disabling NuGet build and test"
Reverting: "Temprary disabling verify-dotnet-consolidation"
Updating NuGet package versions
Updating submodules:
 - Lombiq.ChartJs: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.DataTables: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.HelpfulExtensions: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Hosting.Azure.ApplicationInsights: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Hosting.Tenants: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Privacy: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.UIKit: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.BaseTheme: Updating Lombiq.Tests.UI.* Nuget package version
 - Lombiq.Hosting.MediaTheme: Updating Lombiq.Tests.UI.* Nuget package version

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

# Conflicts:
#	NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj
#	NuGetTest/test/Lombiq.OSOCE.NuGet.Tests.UI/Lombiq.OSOCE.NuGet.Tests.UI.csproj
#	src/Modules/Lombiq.Hosting.Tenants

dministro committed:
Temprary disabling NuGet tests

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

# Conflicts:
#	src/Modules/Lombiq.Hosting.Tenants

dministro committed:
Updating submodule:
 - Lombiq.Hosting.Tenants: Fixing after merge

dministro committed:
Reverting: "Temprary disabling NuGet tests"
Updating NuGet package versions

dministro committed:
Fixing NuGet packages name

dministro committed:
Fixing ShutDownIdleTenants feature name in Lombiq.OSOCE.NuGet.Tests.recipe.json

dministro committed:
Testing that html lang is filled properly under ubuntu

dministro committed:
Reverting "Testing that html lang is filled properly under ubuntu"

dministro committed:
Updating submodule:
 - Lombiq.Privacy: Merge from dev

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions into issue/OSOE-46

sarahelsaig committed:
fast-forward submodules
